examples/getting_started/run.py

```diff
@@ -12,7 +12,100 @@
 import time
 
 
-NUM_STEPS = 100
+NUM_STEPS = 200
+
+
+# Custom Env implementation for OpenBot
+class OpenBotEnv(spear.Env):
+
+    def __init__(self, config):
+    
+        super().__init__(config)
+        self._wheel_rotation_speeds = None
+    
+    def step(self, action):
+
+        obs, reward, is_done, step_info = super().step(action=action)
+        assert "wheel_encoder" in obs.keys()
+        self._wheel_rotation_speeds = obs["wheel_encoder"]
+
+        return obs, reward, is_done, step_info
+
+    def reset(self, reset_info=None):
+    
+        obs = super().reset(reset_info=reset_info)
+        assert "wheel_encoder" in obs.keys()
+        self._wheel_rotation_speeds = obs["wheel_encoder"]
+
+        return obs
+
+    def _apply_action(self, action):
+
+        # calculate wheel torques based on input PWM signals
+        if "apply_voltage" in action.keys():
```

Comment:
`assert `set_duty_cycle` in action.keys()`

Reply:
Updated